Keyword Analysis.ipynb

```diff
@@ -0,0 +1,245 @@
+       "https www instagram      224\n",
+       "instagram                224\n",
+       "instagram com            224\n",
+       "www instagram            224\n",
+       "www instagram com        224\n",
+       "igshid                   222\n",
+       "at barclays              196\n",
+       "police                   186\n",
+       "at barclays center       182\n",
+       "blacklivesmatter         174\n",
+       "blm                      140\n",
+       "army                     132\n",
+       "grand army               128\n",
+       "grand                    128\n",
+       "plaza                    126\n",
+       "grand army plaza         122\n",
+       "army plaza               122\n",
+       "at grand army             94\n",
+       "at grand                  94\n",
+       "georgefloyd               92\n",
+       "center https              90\n",
+       "center https www          90\n",
+       "barclays center https     88\n",
+       "in brooklyn               78\n",
+       "black                     74\n",
+       "floyd                     74\n",
+       "nyc                       72\n",
+       "peaceful                  66\n",
+       "george floyd              66\n",
+       "george                    66\n",
+       "lives                     62\n",
+       "matter                    60\n",
+       "lives matter              60\n",
+       "black lives               60\n",
+       "black lives matter        60\n",
+       "in the                    52\n",
+       "protesters                52\n",
+       "barclayscenter            48\n",
+       "dtype: int64"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "df_BK.iloc[:,1:].sum(axis=0).sort_values(ascending=False)[:50]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# countvectorizing SI tweets\n",
+    "\n",
+    "vectorizer = CountVectorizer(ngram_range=(1,3))\n",
+    "\n",
+    "X = vectorizer.fit_transform(hist_SI['tweet'])\n",
+    "\n",
+    "col_names = vectorizer.get_feature_names()\n",
+    "\n",
+    "col_names.insert(0,'user_id')\n",
+    "\n",
+    "user_ids = pd.DataFrame(hist_SI['user_id'],columns=['user_id'])\n",
+    "\n",
+    "tweets = pd.DataFrame(X.toarray(),columns=vectorizer.get_feature_names())\n",
+    "\n",
+    "# build df with user_ids and countvectorized SI tweets\n",
+    "df_SI = pd.concat([user_ids,tweets],axis=1)\n",
+    "\n",
+    "# create list of stopwords found in the SI tweets\n",
+    "stopwords_SI = list(set(df_SI.columns) & set(stopwords))\n",
+    "\n",
+    "# remove stopwords from countvectorized SI tweets\n",
+    "df_SI = df_SI.drop(stopwords_SI,axis=1)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "police                306\n",
```

Comment:
interesting police appears 100 more times in SI over brooklyn. did you check out any of the tweets w/ the word police in them to see if there are any differences in the content between bk and si?

Reply:
haven't taken a closer look at the tweets themselves, i can check that out today

Reply:
turns out the SI search pulled in over twice as many tweets as the BK search, so that's probably why "police" showed up way more in SI. i think i'll add a tfidf analysis to the the keyword search notebook so that the stats will (hopefully) 
be more comparable between data sets.